.github/workflows/deploy.yml

```diff
@@ -8,27 +8,24 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-latest
-  
+ 
     steps: 
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
     - name: Setup CD Environment
-      uses: yetanalytics/actions/setup-env@v0
-    
+      uses: yetanalytics/actions/setup-env@v0.0.1
+
     - name: Extract version
       id: version
       run: echo ::set-output name=VERSION::${GITHUB_REF#refs\/tags\/v}
 
-    - name: Build JAR file
-      run: make build VERSION=${{ steps.version.outputs.VERSION }}
```

Comment:
Is this still in the makefile?

Reply:
https://github.com/yetanalytics/colossal-squuid/pull/17/commits/7a5dd8adb33f96130d7bbdedeb67909aa5cca32d